### PR TITLE
Add null object

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Comment by vjik** [comment]
Maybe create "in memory session" (save and get data from private properties) instead of "null session"?

**Comment by xepozz** [comment]
You may create it if it is needed

**Comment by vjik** [comment]
> You may create it if it is needed

If create in memory, then null not need

**Comment by xepozz** [comment]
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

**Comment by vjik** [comment]
I write "maybe". I not need both :)

**Review by vjik — CHANGES_REQUESTED** [review]
Need changelog and tests

**Comment by xepozz** [comment]
There is nothing to test

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
 
 ## 1.0.5 under development
 
-- no changes in this release.
+- Enh #39: Add NullSession (@xepozz)
```
Comment (vjik) [inline CHANGELOG.md]:
```suggestion
- New #39: Add `NullSession` (@xepozz)
```

**Comment by vjik** [comment]
> There is nothing to test

Need test get and pull default values. Also can test other methods work without exceptions.
Need tests coverage 100%.

**Review by samdark — APPROVED** [review]

**Pull request merged**